pkg/ccl/backupccl/show.go

```diff
@@ -324,9 +325,16 @@ func showBackupPlanHook(
 		}
 		defer store.Close()
 
+		encStore := store
```

Comment:
Lets add a comment that fleshes out the need for this options.

Reply:
Done